test-common/utils.h

```diff
@@ -193,6 +194,7 @@ struct _EosTestServer
   GObject parent_instance;
 
   GFile *root;
+  Httpd *httpd;
```

Comment:
```suggestion
  Httpd *httpd;  /* (owned) */
```

To make it clearer where ownership lies.

Reply:
Thanks. I do like these GIR style comments you add to structure fields but I always forget to use them myself.